app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -21,6 +21,8 @@ def user_info
     info.merge!(ial2_attributes) if scoper.ial2_scopes_requested?
     info.merge!(x509_attributes) if scoper.x509_scopes_requested?
     info[:verified_at] = verified_at if scoper.verified_at_requested?
+    info[:ial] = Saml::Idp::Constants::AUTHN_CONTEXT_IAL_TO_CLASSREF[identity.ial]
+    info[:aal] = Saml::Idp::Constants::AUTHN_CONTEXT_AAL_TO_CLASSREF[aal]
```

Comment:
**suggestion:** what do you think about moving the default behavior to `OpenidConnectAuthorizeForm#aal`? This way we'll always save an `aal` value in the DB (similar to how the form currently requires an `ial` ACR, so you know there will always be a value in the DB).

Reply:
OIDC Sinatra app doesn't have a value for the. "default" AAL option. So that won't store anything. However, it does save when a value is given - https://github.com/18F/identity-idp/blob/main/app/forms/openid_connect_authorize_form.rb#L83.

The lack of a value is the reason a migration to give a default value to the column was added.

Reply:
Oh wait, I see what you mean. Yeah I can do that.